src/iOS/Xamarin.iOS/Samples/Search/FindPlace/FindPlace.cs

```diff
@@ -30,68 +31,59 @@ namespace ArcGISRuntime.Samples.FindPlace
     /// </summary>
     public class SuggestionSource : UITableViewSource
```

Comment:
Consider move the sample's main class (**FindPlace**) to the top of the file, above supporting classes. That would match the file layout of every other sample.